### ISSUE TITLE
Screen reader is only announcing as Asterisk for "Total number of hours to be billed" field instead of required.

### ISSUE THREAD

**Issue by Kirank915** [microsoft/BotFramework-WebChat#4889]
### Is it an issue related to Adaptive Cards?

No.

### What is the PWD impact?

Screen reader users will not get to know about the required fields.

### What browsers and screen readers do this issue affect?

Windows: Edge with Windows Narrator

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Others or unrelated

### How to reproduce the issue?

1.	Hit the URL https://servicesuat.microsoft.com/#/opdashboard/
2.	Tab till iris bot icon in the header section and press enter.
3.	Iris bit welcome screen will be displayed.
4.	Tab till type your message textbox and type create invoice.
5.	Select the display order number.
6.	Form details for creating invoice will be displayed.
7.	Verify whether the screen reader is only announcing as Asterisk for "Total number of hours to be billed" field instead of required or not.

### What do you expect?

Screen reader should announce the required instead of asterisk for "Total number of hours to be billed" field.

### What actually happened?

Screen reader is only announcing as Asterisk for "Total number of hours to be billed" field instead of required.

**Note:** **This issue repro in both web and desktop app.**

**Environment Details:**
Application Name: Intelligent Support
URL:[Resource360 Dashboard (microsoft.com)](https://servicesuat.microsoft.com/#/opdashboard/)
Chromium Version:114.0.1823.86 (Official build) (64-bit)
Narrator

### Do you have any screenshots or recordings to repro the issue?

https://github.com/microsoft/BotFramework-WebChat/assets/117266902/0c249c28-0c7a-45f0-a69a-0ea8b55ad715

https://github.com/microsoft/BotFramework-WebChat/assets/117266902/3a813a94-f987-49f7-a01f-3352df98093f

### MAS reference

MAS 1.3.1 – Info and Relationships

**Comment by v-prahu** [comment]
#A11yMAS;#A11ySev2;#MAS1.3.1;#HCL;#Accessibility;#FeatureBotframeworkwebchat-Apr23;

**Comment by Kirank915** [comment]
This issue is reported in both web and desktop app and copied from TCS team: [Bug 11092653](https://microsoftit.visualstudio.com/OneITVSO/_workitems/edit/11092653), [Bug 11092659](https://microsoftit.visualstudio.com/OneITVSO/_workitems/edit/11092659)

**Comment by compulim** [comment]
I cannot access the Adaptive Cards.

But looking at the card, I think it's an issue on customer side. This is because the asterisk is black, not red in color.

If the asterisk is red in color, it's added by Adaptive Cards and AC added `aria-hidden` so the screen reader will not narrate "asterisk".

However, in this card, the asterisk appears to be black. That means, the asterisk is added by the customer manually.

![image](https://github.com/microsoft/BotFramework-WebChat/assets/1622400/ecf9fc52-6623-45a3-89fa-3345a0739b33)

**Comment by compulim** [comment]
Closed as customer error.
